dashboard/config/locales/long_instructions.en.yml

```diff
@@ -4895,7 +4895,7 @@ en:
         <strong>How do I add a hyperlink?</strong>
         </summary>
 
-        You can **add hyperlinks to your web page using the `<a>` element**.  The text you want to display as the hyperlink goes between the opening and closing hyperlink tags. The `<a>` element has an attribute `href` which is the location to link to.  The location of a local page is just the file path for that page.
+        You can **add hyperlinks to your web page using the `&lt;a&gt;` element**.  The text you want to display as the hyperlink goes between the opening and closing hyperlink tags. The `&lt;a&gt;` element has an attribute `href` which is the location to link to.  The location of a local page is just the file path for that page.
```

Comment:
I'm not sure what's happening here and whether this is an issue?

Reply:
`&lt;` and `&gt;` are the html entity codes for `<` and `>`, so I suspect the levelbuilder is just making it explicit that this text shouldn't be parsed as html